environment/src/test/java/com/sequenceiq/environment/environment/poller/FreeipaPollerProviderTest.java

```diff
@@ -27,15 +27,15 @@
 import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.describe.DescribeFreeIpaResponse;
 
 @ExtendWith(MockitoExtension.class)
-public class FreeipaPollerCollectionTest {
+public class FreeipaPollerProviderTest {
```

Comment:
JUnit5 does not need public class and public methods. please make them package-private